echopype/tests/echodata/test_echodata.py

```diff
@@ -466,17 +440,68 @@ def test_nan_range_entries(range_check_files):
     assert xr.Dataset.equals(nan_locs_backscatter_r, nan_locs_Sv_range)
 
 
-def test_update_platform(update_platform_samples):
-    raw_file, extra_platform_data_file = update_platform_samples
-    extra_platform_data_file_name = extra_platform_data_file.name
-
-    ed = echopype.open_raw(raw_file, "EK80")
+@pytest.mark.parametrize(
+    ["ext_type", "sonar_model", "updated", "path_model", "raw_path", "platform_data"],
+    [
+        (
+                "external-trajectory",
+                "EK80",
+                ("pitch", "roll", "longitude", "latitude"),
+                "EK80",
+                (
+                        "saildrone",
+                        "SD2019_WCS_v05-Phase0-D20190617-T125959-0.raw",
+                ),
+                (
+                        "saildrone",
+                        "saildrone-gen_5-fisheries-acoustics-code-sprint-sd1039-20190617T130000-20190618T125959-1_hz-v1.1595357449818.nc",  #noqa
+                ),
+        ),
+        (
+                "fixedlocation",
```

Comment:
```suggestion
                "fixed-location",
```
to be consistent with `external-trajectory`